### PR TITLE
Fixed nonzero geometricError on leaf tiles.

### PR DESCRIPTION
**Pull request by nithinp7**
Added a check to set geometricError to 0.0 for all generated leaf tiles. Fixed nonzero geometricError on leaf tiles in hard-coded validation tilesets.

Fixes: #21

### CODE REVIEW

File: CDBTo3DTiles/src/TileFormatIO.cpp [hunk 1]
```diff
@@ -329,14 +328,22 @@ void convertTilesetToJson(const CDBTile &tile, float geometricError, nlohmann::j
         json["content"]["uri"] = *contentURI;
     }
 
-    for (auto child : tile.getChildren()) {
-        if (child == nullptr) {
-            continue;
-        }
+    std::vector<CDBTile*> children = tile.getChildren();
+
+    if(children.empty()) {
```
Comment (lilleyse) [inline CDBTo3DTiles/src/TileFormatIO.cpp:333]: Minor style suggestion. See if you are able to run clang format to auto-format the code. The repo contains a config file in `.clang-format`.

@baothientran how do you normally run clang-format and could those instructions be included in the README?

```suggestion
    if (children.empty()) {
```

**Comment by lilleyse** [comment]
@baothientran can you review?

**Review by baothientran — COMMENTED** [review]
@nithinp7 the change looks good to me. I tested on San Diego CDB and so far all the leaf has geometric error to be 0. I only have a small comment. Once it is fixed, I can merge them

**Comment by baothientran** [comment]
thanks @nithinp7, I tested with other dataset and it works quite well

**Pull request merged**